frontend/packages/dev-console/integration-tests/features/web-terminal/web-Terminal-adminuser.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Web Terminal for Admin user
+    As a user with admin rights, I should be able to use web terminal
+
+
+Background:
+    Given user has logged in as admin user
+    And user is at developer perspecitve
+    And user has installed Web Terminal operator
+    And user has created Web Terminal instance
```

Comment:
What is a web terminal instance?

Reply:
It was referring to 'Create Web Terminal instance' scenario. But I have removed this scenario so I will remove that statement as well.